airflow/providers/slack/CHANGELOG.rst

```diff
@@ -33,6 +36,27 @@ Breaking changes
 * Drop support deprecated ``webhook_token`` parameter in
   :ref:`Slack Incoming Webhook Connection <howto/connection:slack-incoming-webhook>` Extra.
 
+* ``Refactor 'SlackWebhookOperator': Get rid of mandatory http-provider dependency (#26648)``
+* ``Refactor SlackWebhookHook in order to use 'slack_sdk' instead of HttpHook methods (#26452)``
+* ``Move send_file method into SlackHook (#26118)``
+
```

Comment:
#26118 it is more like feature, previously code which send file to slack exists only in specific Operator, this code moved in Hook, so user might now use in their own callbacks/PythonOperators

Reply:
Good point!. Thanks for comments @Taragolis